frontend/src/components/RegistrationForm.vue

```diff
@@ -1,86 +1,144 @@
-                        <li>At least {{ fieldValidations.ACRONYM_MIN_LENGTH }} characters</li>
-                        <li>
-                          Examples:
-                          <em>ABCD</em>,
-                          <em>ABCD_WXYZ</em>
-                        </li>
-                      </ul>
-                    </v-tooltip>
-                  </template>
+                  <template v-slot:append-outer></template>
                 </v-text-field>
               </v-col>
             </v-row>
-
-            <v-textarea v-model="comments" rows="1" auto-grow label="Comments"></v-textarea>
+            <v-row>
+              <v-col>
+                <label>Comments (optional)</label>
+                <v-textarea
+                  v-model="comments"
+                  rows="3"
+                  auto-grow
+                  hide-details="auto"
+                  dense
+                  solo
+                  outlined
+                  flat
+                ></v-textarea>
+              </v-col>
+            </v-row>
           </v-form>
         </v-card-text>
-        <v-card-actions>
-          <v-spacer></v-spacer>
-          <v-btn text @click="cancel()">Cancel</v-btn>
-          <v-btn color="success darken-1" text @click="postRegistrationForm()">SEND</v-btn>
+        <v-card-actions class="justify-center pb-8">
+          <v-btn class="BC-Gov-PrimaryButton light" text @click="cancel()">Cancel</v-btn>
+          <v-btn class="BC-Gov-PrimaryButton" text @click="postRegistrationForm()">Submit</v-btn>
         </v-card-actions>
       </v-card>
     </v-dialog>
-  </v-btn>
-</template>
 
+    <Dialog v-bind:show="errorOccurred" @close-dialog="errorOccurred = false">
+      <template v-slot:icon>
+        <v-icon large color="red">cancel</v-icon>
+      </template>
+      <template v-slot:text>
+        <p>
+          An error occurred while attempting to add your application.
+          <br />You can also add your application by sending an email to
+          <a
+            href="mailto:NR.CommonServiceShowcase@gov.bc.ca?subject=GETOK Registration for <acronym> - <idir>"
+          >NR.CommonServiceShowcase@gov.bc.ca</a>
+        </p>
+        <p>Please include your Acronym as well as your IDIR username in your email.</p>
+      </template>
+    </Dialog>
 
+    <Dialog v-bind:show="registerSuccess" @close-dialog="registerSuccess = false">
+      <template v-slot:icon>
+        <v-icon large color="green">check_circle_outline</v-icon>
+      </template>
+      <template v-slot:text>
+        <p>Your Registration was sent successfully. You will be sent an email to ${this.userInfo.emailAddress} when your application has been authorized.</p>
+      </template>
+    </Dialog>
+  </v-btn>
+</template>
 <script>
 import ApiService from '@/common/apiService';
 import { FieldValidations } from '@/utils/constants.js';
 import { mapGetters } from 'vuex';
+import '@/assets/scss/style.scss';
```

Comment:
We'll need to consider how this is going to be "globalized" for bcgov styles. This will work for now as a shim however.